src/page-multi-facility/types.tsx

```diff
@@ -64,3 +64,32 @@ export type ScenarioUsers = {
   owner: User | null | undefined;
   viewers: User[];
 };
+
+type SimpleTimeseries = {
+  date: Date;
+  value: number;
+};
+
+export type ShadowCovidCase = {
+  observedAt: Date;
+  popTested?: number;
+  popTestedPositive?: number;
+  popTestedNegative?: number;
+  popDeaths?: number;
+  staffTested?: number;
+  staffTestedPositive?: number;
+  staffTestedNegative?: number;
+  staffDeaths?: number;
+};
+
+export type ShadowFacility = {
+  id: string;
+  state: string;
+  canonicalName: string;
+  facilityType: string;
+  capacity: SimpleTimeseries[];
+  population: SimpleTimeseries[];
+  covidCases: ShadowCovidCase[];
+  // various other metadata that we don't explicitly care about may also be present
+  [key: string]: unknown;
+};
```

Comment:
I think you know this, but just as an FYI, the only data we're going to have out of the gate for reference facilities are:

```js
state
canonicalName
facilityType
covidCases
```

We won't have the other data (i.e. `capacity`, `population`, etc.) until we're at a point where we can import metadata from BigQuery.

That said, I think it's fine to leave these in for now as they represent the idealized future shape of these documents.

Reply:
yes that's part of the reason why I want to keep using the test collection for now ... although while this typedef does require those fields to be present, it wouldn't object to those arrays being empty. I am assuming that is a case we are going to have to handle in the real world as well